dbtwiz/manifest.py

```diff
@@ -35,18 +35,34 @@ def models_cached(cls):
 
 
     @classmethod
-    def update_manifests(cls):
-        """Rebuild local manifest and download latest production manifest"""
+    def update_manifest(cls):
```

Comment:
Rename to `rebuild_manifest` to distinguish name more from the new `update_manifests` method?

Reply:
Agreed!